pom.xml

```diff
@@ -111,20 +122,9 @@
                 <spark.version>${spark302.version}</spark.version>
                 <spark.test.version>${spark302.version}</spark.test.version>
             </properties>
-        </profile>
-        <profile>
-            <id>release301emr</id>
```

Comment:
I thought we weren't going to pull the EMR shim out in this PR?

Reply:
ok put it all back so the default build now contains it.